### ISSUE TITLE
[DOC] Group documents into clusters using Cluster layer

### ISSUE THREAD

**Issue by VijayanB** [opensearch-project/documentation-website#3987]
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [x] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
Add cluster layer to Maps documentation.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/dashboards-maps/issues/250

**Comment by hdhalter** [comment]
@vagimeli , @kolchfa-aws - Fanit is assigned to the doc issue, but Melissa is assigned to the github issue. Who is covering this one? Thanks!

**Comment by kolchfa-aws** [comment]
@hdhalter I'm ok with any option. I'll discuss with @vagimeli.

**Comment by vagimeli** [comment]
@kolchfa-aws +1

**Comment by vagimeli** [comment]
@hdhalter Assign the issue to me. cc: @kolchfa-aws

**Comment by hdhalter** [comment]
Moved to 2.11, as per project meeting 8/10.

**Comment by vagimeli** [comment]
@VijayanB Will you send me the demo and endpoint I can use to finish drafting the documentation for this feature? Once the draft content is ready, I'll send to you for technical review. Thanks!

**Comment by vagimeli** [comment]
Per the roadmap, this feature is for 3.0.0

**Comment by vagimeli** [comment]
@VijayanB Is this issue still on hold?

**Comment by vagimeli** [comment]
@VijayanB Can we close this issue and open a mew issue or PR when the feature needs documentation?  

cc: @hdhalter

**Comment by dblock** [comment]
[Triage -- attendees [1](https://github.com/dblock), [2](https://github.com/peternied), [3](https://github.com/reta), [4](https://github.com/krisfreedain), [5](https://github.com/Getsaurabh02), [6](https://github.com/gkamat), [7](https://github.com/scrawfor99)]
